docs/elements/hx-tile/index.html

```diff
@@ -0,0 +1,28 @@
+---
+title: <hx-tile>
+also:
+  components/choice-tiles: Choice Tiles
+  elements/hx-tile-title: <hx-tile-title>
+  elements/hx-tile-description: <hx-tile-description>
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+      The custom <code>&lt;hx-tile&gt;</code> element is compromised of an icon,
+      label, and optional description.
+    </p>
+
+     <hx-dl class="hxBox-md metadata">
+      <hx-def>
+        <hx-dt>Permitted Parents</hx-dt>
+        <hx-dd>any</hx-dd>
+      </hx-def>
+      <hx-def>
+        <hx-dt>Permitted Children</hx-dt>
+        <hx-dd>
```

Comment:
Are there any permitted children? Or should `none` be in here? It just seems a bit odd as blank to me.

Reply:
good catch! pushed some changed to update those docs